### PR TITLE
Express answers to survey

### PR DESCRIPTION
**Pull request by wesleytodd**
I know it has been slow in getting to these (which is why we need help), but here they are.  Let me know if there are any areas we can answer better.

### CODE REVIEW

**Comment by Eomm** [comment]
Could we try to define some tasks or we will do it in a separate PR?

File: pilots/express/SURVEY_ANSWER.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Express answers to the survey
+
+## What are the biggest problems your package need to face?
```
[the remark this answers is not in the document]
Comment (Qix-) [reply pilots/express/SURVEY_ANSWER.md]: As is, the grammar is a bit odd.

**Comment by mhdawson** [comment]
@Eomm, ideally we should document the current 'top 10' in an issue (either here or in the express repo) and have at least a pointer from this repo so that we can actively track what we should encouraging people to help out with.

File: pilots/express/SURVEY_ANSWER.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (wesleytodd) [reply pilots/express/SURVEY_ANSWER.md]: This the the phrasing presented in the survey questions.  Maybe after we merge this, @Qix-, you could contribute a PR to fix it in all three places?  The survey, this doc, and the answers from MQTT.js?

**Comment by wesleytodd** [comment]
When I get home from the office today I will extract the list I have and post it as an issue in the Express discussions repo.  I will post a link here for reference and we can pull out tasks from there.

File: pilots/express/SURVEY_ANSWER.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mhdawson) [reply pilots/express/SURVEY_ANSWER.md]: @wesleytodd that makes sense to me.

**Comment by mhdawson** [comment]
Unless there are objections I'll plan to land this in the next day or two.

**Pull request merged**